docs/design/storagenode-graceful-exit.md

```diff
@@ -36,31 +36,32 @@ This process including the Storage nodes transferring their pieces to other node
-- When a Storage node triggers graceful exit I want them to be omitted from the node selection process for uploads so they are no longer selected to store new pieces.
-- When a Storage node triggers graceful exit I want their allocated bandwidth on the network to be ignored so that they can complete their graceful exit as quickly as possible.
+- When a Storage node no longer wants to store data for a satellite, it shall have the ability to run graceful exit for a specific satellite so that they do not lose their escrow on that satellite. 
+- When a Storage Node runs the graceful exit command, it shall be prompted with a confirmation message so that it can avoid running the command accidentally.
+- When a storage node triggers the graceful exits, the process cannot be canceled. The storage node shall lead it to successful completion. Should the storagenode application, ungracefully terminated, upon restarting the storagnenode application, it shall resume the graceful exit process.
+- When a Storage node triggers graceful exit, the satellite shall omit it from selecting the node for any future uploads and shall be eliminated from storing new pieces.
+- When a Storage node triggers graceful exit, it shall ignore the allocated network bandwidth, so that it shall complete their graceful exit as quickly as possible.
 
 ### During exit
-- When a Storage Node is in the process of gracefully exiting the network I want them to continue to be audited and their uptime checked so we can ensure they are still "good" nodes.
-- When A Storage Node is in the process of gracefully exiting the network I want them to be selected for download requests so that they can continue to contribute to the overall network.
-	- we need to be able to distinguish the bandwidth used for serving up data to clients vs bandwidth used for graceful exit. the node will NOT get paid for bandwidth used for graceful exit but will get paid for the bandwidth used to serve data to clients.
-- When a Storage Node is gracefully exiting the network I want all of the pieces they are storing to be deleted from their hard drive as they exit so that their computers hard drive space is no longer used.
-- When a Storage Node is exiting the network gracefully I want the satellite to have the ability to track how much egress they used for exiting so that we do not pay them for that bandwidth.
+- When a Storage Node is in the process of gracefully exiting the network, it shall continue to participate in the requested audits and uptimes checks.
+- When a Storage Node is in the process of gracefully exiting the network, it shall continue to honor to download requests.
+	- The Storage Node keep a separate and detailed metric of network bandwidth used to serve the data to clients vs bandwidth used for graceful exit. The Storage Node shall NOT get paid for bandwidth used for graceful exit but shall get paid for the bandwidth used to serve data to clients(downloads, audits, uptime checks etc...)
```

Comment:
I don't think we are paying for `audits, uptime checks etc...`

Reply:
For sure, we are not paying for uptime checks. We should double-check if we are paying for audits.

Reply:
updated to downloads and audit egress

Reply:
> For sure, we are not paying for uptime checks. We should double-check if we are paying for audits.

We are paying for audits traffic.